tests/Stripe/CustomerTest.php

```diff
@@ -6,6 +6,7 @@ class CustomerTest extends TestCase
 {
     const TEST_RESOURCE_ID = 'cus_123';
     const TEST_SOURCE_ID = 'ba_123';
+    const TEST_TAX_ID = 'txi_123';
```

Comment:
Should maybe be `TEST_TAX_ID_ID` since `tax_id` is the name of the resource, but that's silly :p